zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirector.java

```diff
@@ -407,11 +407,8 @@ protected void onActorClosed() {
   @Override
   protected void onActorCloseRequested() {
     isOpened.set(false);
-    if (exporterMode == ExporterMode.ACTIVE) {
-      containers.forEach(ExporterContainer::close);
-    } else {
-      exporterDistributionService.close();
-    }
+    containers.forEach(ExporterContainer::close);
```

Comment:
:wrench: `close` may throw, we should make sure that we call close on everything neverthless

Reply:
`ExporterContainer#close` doesn't throw, it logs all errors and doesn't rethrow them.